src/quest/score-quest.js

```diff
@@ -0,0 +1,9 @@
+import quests from './services/quest-data';
```

Comment:
Double check this file path, you're getting a 404 error because it can't find the file